README.md

```diff
@@ -7,34 +7,30 @@ Rails
 Items
 =====
 
- - **Rails**
- Rails is the most fundamental part of this module. It is the track in which locomotives and minecarts can travel on.
- 
- - **Locomotive**
- Locomotive (Steam Engine) is a rail transport vehicle that provides the motive power for a minecart. Locomotive only works on rails. You can switch the locomotives on/off by pressing `e` while facing the locomotive. You can't push a locomotive, it will stay stationary unless you turn it on. You can destroy locomotives by left-clicking while facing the locomotive.
- 
- - **Minecart**
- Minecart is a rail transport vehicle that can be used by the player to travel far distances faster. You can ride the minecart by pressing `e` while facing the minecart. Although minecart can move without a locomotive by pressing `w` to move forward and pressing `s` to stop, using locomotive is better because it gives a significant boost in speed. Minecart will change direction if it hits an obstacle (Ex: Unlinked Locomotive, Block). You can't push a minecart, it will stay stationary unless you ride on it or a moving locomotive is linked to it. You can destroy minecart by left-clicking while facing it.
- 
- - **Wrench**
- Wrench is a tool that can be used to connect minecarts with a locomotive and also change the direction of a tee joint.
+## Rails
+Rails are the most fundamental part of this module. They are the tracks on which minecarts can travel.
```

Comment:
Can we replace the word `minecart` with `cart` over here? From what I've seen, we usually use plain ol' `carts`, including in other parts of this readme.